Makefile

```diff
@@ -132,6 +132,11 @@ e2e-setup-kind: setup-local-registry
 	./scripts/connect-registry.sh &
 	sleep 90s
 
+e2e-kmsv2-setup-kind: setup-local-registry
+	./scripts/setup-kmsv2-kind-cluster.sh &
+	./scripts/connect-registry.sh &
+	sleep 90s
```

Comment:
is it possible to watch/poll something instead of hardcoded sleep?

Reply:
can we run both async and wait for pids to finish? https://github.com/open-policy-agent/gatekeeper/blob/master/.github/workflows/release.yaml#L110-L120

Reply:
When we run the process in the background ADO marks the task as completed

The error is:
```shell
The STDIO streams did not close within 10 seconds of the exit event from process '/bin/bash'. This may indicate a child process inherited the STDIO streams and has not yet exited.
```
 
I think this is the reason we have sleep, to hold on to the task for enough time so that the registry can be connected to the Kind cluster and created successfully. 

@aramase  to add any additional context.